Analysis/TrueJet/src/TrueJet.cc

```diff
@@ -698,6 +704,12 @@ void TrueJet::processEvent( LCEvent * event ) {
         // post-PS part
 
       LCCollectionVec* fafpf_vec = new LCCollectionVec(LCIO::RECONSTRUCTEDPARTICLE ) ;
+      auto fafpfPidHandler = UTIL::PIDHandler(fafpf_vec);
+      auto fafpf_mainPidId = fafpfPidHandler.addAlgorithm("TrueJet_fafpf", {});
+      std::array<int, 25> fafpf_pidIds{};
+      for (size_t i = 0; i < fafpf_pidIds.size(); ++i) {
+          fafpf_pidIds[i] = fafpfPidHandler.addAlgorithm("TrueJet_fafpf_jet_" + std::to_string(i), {});
```

Comment:
```suggestion
          fafpf_pidIds[i] = fafpfPidHandler.addAlgorithm("TrueJet_fafpf_jet_" + std::to_string(i + 1), {});
```
> Also the 0'th one is never filled? The files become really crowded at the moment with all of them in there...

Reply:
doesn't this do the opposite? I.e. it just renames 0 to 1 but it will still never be filled?

Reply:
It doesn't fix that, but I just remembered I wanted to do this, because previously the one that goes currently into the "main" PID went into `0`.